mkdocs.yml

```diff
@@ -0,0 +1,66 @@
+extra_css:
+  - assets/styles/extra.css
+
+site_name: DiracX
+repo_url: https://github.com/DIRACGrid/diracx
+site_url: https://diracx.io/
```

Comment:
I'd put `https://diracgrid.org` here

Reply:
But that isn't where this will be hosted unless you're thinking of dropping the old site?